tests/background/sidebar-injector-test.js

```diff
@@ -436,7 +436,7 @@ describe('SidebarInjector', () => {
         try {
           await injector.injectIntoTab({
             id: 1,
-            url: 'https://qa-lms.ca.hypothes.is/lti_launches',
+            url: 'https://staging-lms.ca.hypothes.is/lti_launches',
```

Comment:
As per my previous comment, I think this URL might not be correct, but I can't find what would be the right one, if `lms.staging.ca.hypothes.is` or `lms.ca.staging.hypothes.is`